cime_config/allactive/config_compsets.xml

```diff
@@ -320,17 +320,27 @@
 
 <compset>
   <alias>CRYO1850</alias>
-  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
+  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
 <compset>
   <alias>CRYO1850-4xCO2</alias>
-  <lname>1850SOI_EAM%CMIP6-4xCO2_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
+  <lname>1850SOI_EAM%CMIP6-4xCO2_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
 <compset>
   <alias>CRYO1950</alias>
-  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBISMF_MOSART_SGLC_SWAV</lname>
+  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBPISMF_MOSART_SGLC_SWAV</lname>
+</compset>
+
+<compset>
+  <alias>CRYO1850-DISMF</alias>
+  <lname>1850SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBDISMF_MOSART_SGLC_SWAV</lname>
+</compset>
+
+<compset>
+  <alias>CRYO1950-DISMF</alias>
+  <lname>1950SOI_EAM%CMIP6_ELM%SPBC_MPASSI%DIB_MPASO%IBDISMF_MOSART_SGLC_SWAV</lname>
 </compset>
 
 <compset>
```

Comment:
@xylar there are three more `CRYO` compsets that still have the `IBISMF` tag in the compset long name. These were brought in https://github.com/E3SM-Project/E3SM/pull/5616, which was only merged a month ago, so these changes probably snuck in when you rebased.

Reply:
Sorry, yes, you're right that I missed that when I rebased.  Thanks for catching those!

Reply:
Should be fixed.